chcemvediet/locale/sk/LC_MESSAGES/django.po

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-05-02 16:58+0200\n"
+"POT-Creation-Date: 2021-05-18 11:38+0200\n"
```

Comment:
`makemessages` robil "bordel", tak som to pridal rucne. [Sortovanie](https://code.djangoproject.com/ticket/29734) bolo pridane az neskor.

Reply:
Ok. Mozeme sa dohodnut, ze proste `msgid` budu podla abecedy a budeme ich pridavat rucne. Potom ich vsak pridaj v poradi podla abecedy.